models/marts/gender_avg_satisfaction_scores.sql

```diff
@@ -0,0 +1,56 @@
+WITH satisfaction_data AS (
```

Comment:
can you move this cte to staging folder. Please always pull relevant info in staging and build on top of it.